scripts/sql/91_create_chart_metadata.down.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE "public"."chart_ref_metadata" CASCADE;
+
+----Rows Deletion---
+DELETE FROM "public"."chart_ref_metadata" WHERE ("chart_name"= 'Rollout Deployment')
```

Comment:
remove delete statements. as this will throw error as you have dropped the table just above